src/cfnlint/config.py

```diff
@@ -305,7 +305,16 @@ def __call__(self, parser, namespace, values, option_string=None):
                         items.append(value)
                 setattr(namespace, self.dest, items)
 
-        parser = ArgumentParser(description='CloudFormation Linter')
+        usage = (
+            '\nBasic: cfn-lint test.yaml\n'
+            'Ignore a rule: cfn-lint -I E3012 -- test.yaml\n'
+            'Configure a rule: cfn-lint -x E3012:strict=false -t test.yaml\n'
+            'Lint all yaml files in a folder: cfn-lint dir/**.yaml'
```

Comment:
Go for the recursive example maybe?

`cfn-lint dir/**/*.yaml`

Reply:
Fixed.